static/js/components/forms/ChangeEmailForm.js

```diff
@@ -47,19 +47,27 @@ const ChangeEmailForm = ({ onSubmit, user }: Props) => (
             <label htmlFor="email">Email</label>
             <Field
               name="email"
+              id="email"
               className="form-control"
               component={EmailInput}
+              aria-describedby="email_error"
             />
-            <ErrorMessage name="email" component={FormError} />
+            <ErrorMessage name="email" id="email_error" component={FormError} />
           </div>
           <div className="form-group">
             <label htmlFor="confirmPassword">Confirm Password</label>
             <Field
+              id="confirmPassword"
               name="confirmPassword"
               className="form-control"
               component={PasswordInput}
+              aria-describedby="confirm_password_error"
```

Comment:
Sorry, I don't think I was clear enough in my last review. I was suggesting that only the fields that already have the anti-pattern snake case attribute values (like `name="profile.job_title"`) should have ids that are also snake case. I know it seems weird to have different patterns for the same attributes in different files, but eventually we're going to refactor those snake case attribute values (#186) and the id's/aria attributes can be updated as well. If you notice here, `confirm_password_error` introduces **three** different conventions for attribute values in a single component. We have camel case for `id` and `name`, lowercase + dashes for `className`, and now snake case for `aria-describedby`. Dashes in CSS class names are already an understood convention that is specific to CSS. For other attributes, we should try to stick to camel case. So for example:

- ~~`aria-describedby="confirm_password_error"`~~ --> `aria-describedby="confirmPasswordError"`
- ~~`aria-describedby="email_error"`~~ --> `aria-describedby="emailError"`

Please apply that logic to all `id` and `aria-describedby` attributes in this PR _except for_ the ones that are associated with a field that already has snake case attribute values (like `legal_address.first_name`)

Reply:
This was my first thought as well However `please apply the same naming scheme to these other attributes that were added in this PR.` this line threw me off a bit and I assumed I had to apply the style to all attributes to maintain consistency.
I should have consulted you before hand, that was an oversight on my end and as requested above I have completed the required change.